Mars_lander.py

```diff
@@ -0,0 +1,161 @@
+"""MVC mars lander game
+Land dragon on Mars with arrow keys"""
+
+import pygame
+import math
+
+WindowWidth = 1366  # width of the program's window, in pixels
+WindowHeight = 768  # height of the program's window, in pixels
+
+class Lander(object):
+   """Model of Lander, with attributes, X,Y,Rotation,Fuel,dX, and dY
+   and methods gravity_update ,roll, and fire_thrusters"""
+   def __init__(self):
+      self.X = 100  #Lander X coordinate in Pixels
```

Comment:
Generally, it's stylistically better to name your class attributes beginning with a lowercase letter. Generally, it's convention to name instances of a class with a lowercase letter, and name the class itself with an uppercase letter. In this case, X, Y, Fuel, etc. are all instances and not a class themselves, so they should all be lowercase